tests/extension/req-abort-missing-custom-json-template.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Abort request as a result of rinit, with a non-existent template
+--INI--
+datadog.appsec.enabled=1
+datadog.appsec.log_level=error
+--ENV--
+DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON=tests/extension/templates/missing
+--FILE--
+<?php
+use function datadog\appsec\testing\rinit;
+
+include __DIR__ . '/inc/mock_helper.php';
+
+$helper = Helper::createInitedRun([
+    response_list(response_request_init(['block', ['status_code' => '500', 'type' => 'json'], ['{"found":"attack"}','{"another":"attack"}']])),
```

Comment:
Since you have the logic to handle it, I would like to see a test where the helper response with more extra fields apart of `status_code` and `type`

Reply:
I'll add a few random ones, although they should just be ignored (not sure if you're expecting something else to happen).

Reply:
No no, it is just to ensure that the logic behind ignoring those fields keep working in case it gets changed